packages/hint/docs/contributor-guide/getting-started/events.md

```diff
@@ -236,5 +225,6 @@ type CanEvaluateScript {
 <!-- Link labels: -->
 
 [nodeName docs]: https://developer.mozilla.org/en-US/docs/Web/API/Node/nodeName
+[parsers]: https://webhint.io/docs/user-guide/concepts/parsers/
 [postcss]: https://postcss.org/
```

Comment:
can you double check if this link is still used in this file? It looks like it was part of the `parse::css` section.

Reply:
Good catch, thanks!